src/Sulu/Bundle/SecurityBundle/Command/CreateRoleCommand.php

```diff
@@ -81,7 +81,7 @@ function ($value) use (&$securityContextsFlat) {
             $permission = new Permission();
             $permission->setRole($role);
             $permission->setContext($securityContext);
-            $permission->setPermissions(120);
+            $permission->setPermissions(121);
```

Comment:
I'll set it to `127` for now. As long as there are only 7 permissions this should be enough, and it's not really part of this PR...